.gitignore

```
@@ -16,5 +16,5 @@ node_modules/
 .work/
 
 # Crossplane package
-/package/crossplane.yaml
+crossplane.yaml
```

there is no top-level yaml file
```suggestion
/package/crossplane.yaml
```

---

Yeah just fixed that with a new push :)